sections/home-hero-section/index.js

```diff
@@ -0,0 +1,139 @@
+import { rem } from 'polished'
+import Typist from 'react-typist'
+import styled from 'styled-components'
+
+import { ApplyLink, Heading, Text } from 'components'
+import 'react-typist/dist/Typist.css'
+import { TABLET_LARGE_SIZE, MOBILE_SIZE } from 'styles/constants'
+import styles from 'styles/Home.module.css'
+
+const StyledSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 144px;
```

Comment:
There are a few sizes here that are not very consistent with other sections. Could you please review the designs one more time? I might be missing something, the designs seem different than the result.

![Screen Shot 2021-04-12 at 3 20 09 PM](https://user-images.githubusercontent.com/832191/114469764-9747bb80-9ba2-11eb-974e-6b65448d7ba5.png)

Reply:
Another note, try to limit the custom numbers used in the styles. Right now this extra padding of 144px seems unnecessary and makes it harder for me to understand or change the styles if needed.

Instead, I can see that in Figma
- the width of this text region on desktop is 782px
- the horizontal padding of the region on mobile is 36px

So you can write that with no media queries:
```
max-width: ${rem(782 + 36 * 2)};
padding: 0 {rem(36)};
margin: 0 auto;
```

Reply:
There's still a lot of magic numbers in the styles. The Do What's Right component in particular is a very custom layout.